File: 3194-find-words-containing-character/3194-find-words-containing-character.py
class Solution(object):
    def findWordsContaining(self, words, x):
        """
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """
        lst = []
        k=0
        for i in words:
            if x in i:
                lst.append(k)
            k += 1
        return lst

        

File: 3194-find-words-containing-character/3194-find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> arr = new ArrayList<>();
        for(int i=0;i<words.length;i++){
           if(words[i].indexOf(x) != -1)
           arr.add(i);
        }
    return arr;    
    }
}